cdci_data_analysis/analysis/parameters.py

```diff
@@ -520,7 +521,8 @@ def from_owl_uri(cls,
                 if python_subclass.matches_owl_uri(owl_superclass_uri):
                     logger.info("will construct %s by owl_uri %s", python_subclass, owl_superclass_uri)
                     call_kwargs = {}
-                    call_signature = signature(python_subclass)                   
+                    call_signature = signature(python_subclass)
+                    var_kw_signature = call_parameter.VAR_KEYWORD in [x.kind for x in call_signature]
```

Comment:
can you actually loop over the signature object?

According to https://docs.python.org/3.8/library/inspect.html#inspect.signature

you have to use `call_signatures.parameters.values()`

```suggestion
                    var_kw_signature = call_parameter.VAR_KEYWORD in [x.kind for x in call_signature.parameters.values()]
```

Reply:
You are right, thank you. It's dict-like. 
I wrote it a bit in a hurry, will fix